Assets/Scripts/UI/SkillUI.cs

```diff
@@ -0,0 +1,19 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class SkillUI : MonoBehaviour
+{
+	protected StatUIIconTag _icon = null;
+	protected PlayerStats _playerStats = null;
+
+	public virtual void SetStat( float stat ) { } 
+	public virtual void SetMaxStat( float stat ) { }
```

Comment:
I think I did that because not every stat UI is going to have a use for both calls. E.g. skillBubbles doesn't use SetStat()